### PR TITLE
Added new test cancel reset test after boot

### PR DESCRIPTION
**Pull request by lalon4**
also aligned to PEP8

### CODE REVIEW

**Comment by nshidlin** [comment]
LGTM

File: discovery-infra/utils.py [hunk 1]
```diff
@@ -267,6 +267,45 @@ def wait_till_at_least_one_host_is_in_status(
...
+            lambda: are_host_progress_in_stage(
+                client.get_cluster_hosts(cluster_id),
+                stages,
+            ),
+            timeout_seconds=timeout,
+            sleep_seconds=interval,
+            waiting_for="Node to be in of the stage %s" % stages,
+        )
+    except:
+        hosts = client.get_cluster_hosts(cluster_id)
```
Comment (tsorya) [inline discovery-infra/utils.py:289]: lets have as function in teardown

Comment (nshidlin) [reply discovery-infra/utils.py:289]: Why should this be in teardown?

**Review by YuviGold — APPROVED** [review]
LGTM

File: discovery-infra/tests/base_test.py [hunk 2]
```diff
@@ -1,32 +1,33 @@
...
-        yield controler
-        controler.shutdown_all_nodes()
-        controler.format_all_node_disks()
-
-    def create_cluster(self, api_client):
+    def node_controller(self):
+        controller = nodeController()
+        yield controller
+        controller.shutdown_all_nodes()
+        controller.format_all_node_disks()
```
Comment (tsorya) [inline discovery-infra/tests/base_test.py:27]: we will need to add bring all logs here

**Pull request merged**